src/components/diagram-2a-chart-card.js

```diff
@@ -1,17 +1,17 @@
-import React, { Fragment } from "react"
+import React from "react"
 
 const Chart2aCard = (props) => (
-  <Fragment>
-    <dt className="util-relative util-zindex-100 cmp-diagram-chart__container" style={{["--percent"]: props.percent + "%", ["--color"]: `var(--color-${props.color})`}}>
+  <div className="cmp-diagram-chart__item">
+    <dt className="util-relative util-zindex-100 cmp-diagram-chart__value-container" style={{"height": props.percent * 2 + "%"}}>
```

Comment:
Unless this is a one off use, these utility class properties ought to be a part of the `__value-container` class.

Reply:
Gotcha, added that to the CSS 👍